src/components/base/Tag/Tag.module.scss

```diff
@@ -0,0 +1,26 @@
+@use "../../../styles/mixins/mixins-typo" as *;
+@use "../../../styles/utilities/functions" as *;
+@use "../../../styles/utilities/utilities" as *;
+
+.tag {
+  @include label-text;
+
+  --spacing-block: 6;
+
+  display: inline-block;
+  padding: px-to-rem(var(--spacing-2xs)) px-to-rem(var(--spacing-block));
+
```

Comment:
I know that the vertical alignment is off in Figma. I would try to vertically center this with

```
@supports (text-box-trim: trim-both) {
    text-box-edge: cap alphabetic;
    text-box-trim: trim-both;
    padding-block: px-to-rem(var(--spacing-xs));
  }
```

Still not perfect everywhere but more pleasent to my eye.